packages/block-library/src/navigation/edit/index.js

```diff
@@ -297,6 +298,14 @@ function Navigation( {
 		[ clientId ]
 	);
 	const isResponsive = 'never' !== overlayMenu;
+	const isMobileBreakPoint = useMediaQuery(
+		`(max-width: ${ NAVIGATION_MOBILE_BREAKPOINT }px)`
+	);
+
+	const isCollapsed = () =>
+		( 'mobile' === overlayMenu && isMobileBreakPoint ) ||
+		'always' === overlayMenu;
```

Comment:
```suggestion
	const isCollapsed = ( 'mobile' === overlayMenu && isMobileBreakPoint ) ||
		'always' === overlayMenu;
```

Can this become a variable?